corehq/apps/app_manager/forms.py

```diff
@@ -18,20 +19,31 @@
 from .dbaccessors import get_all_built_app_ids_and_versions
 from .models import LATEST_APK_VALUE, LATEST_APP_VALUE
 from .util import get_commcare_builds
+from ..accounting.utils import domain_has_privilege
+from ..hqwebapp.widgets import BootstrapCheckboxInput
+from ...privileges import RELEASE_MANAGEMENT
 
 
 class CopyApplicationForm(forms.Form):
     domain = forms.CharField(
         label=ugettext_lazy("Copy this app to project"),
         widget=forms.Select(choices=[], attrs={
-            "data-bind": "autocompleteSelect2: domain_names",
+            "data-bind": "autocompleteSelect2: domainNames, event:{ change: domainChanged }",
         }))
     name = forms.CharField(required=True, label=ugettext_lazy('Name'))
     linked = forms.BooleanField(
         required=False,
         label=_('Copy as Linked Application'),
+        widget=BootstrapCheckboxInput(
+            inline_label=mark_safe(_(
+                "<!-- ko ifnot: shouldEnableLinkedAppOption -->"
+                "Cannot link this application with the selected project space."
```

Comment:
Could this say **why** app can't be linked?

Reply:
Yeah I thought about this. I kept it vague because I thought about it as 2 separate reasons. One being that the domain isn't part of the enterprise account, and the other being it is part of the enterprise account but isn't link. In hindsight, saying the domain isn't linked is sufficient, and they can figure out via the domain linking interface/documentation that they can only link to domains within the enterprise account. 

I'll change it to "The selected project space must first be linked to the current project space." for now, and bring this up to product.